source/crud/crud.rst

```diff
@@ -1577,9 +1567,13 @@ Q: Where did modifiers go in FindOptions?
 Q: Where is ``save``?
   Drivers have historically provided a ``save`` method, which was syntactic sugar for upserting or inserting a document based on whether it contained an identifier, respectively. While the ``save`` method may be convenient for interactive environments, such as the shell, it was intentionally excluded from the CRUD specification for language drivers for several reasons. The ``save`` method promotes a design pattern of "fetch, modify, replace" and invites race conditions in application logic. Additionally, the split nature of ``save`` makes it difficult to discern at a glance if application code will perform an insert or potentially dangerous full-document replacement. Instead of relying on ``save``, application code should know whether document already has an identifier and explicitly call ``insertOne`` or ``replaceOne`` with the ``upsert`` option.
 
+Q: Where is ``useCursor`` in AggregateOptions?
+  Inline aggregation results are no longer supported in server 3.5.2. The `aggregate command <https://docs.mongodb.com/manual/reference/command/aggregate/>`_ must be provided either the ``cursor`` document or the ``explain`` boolean. Since AggregateOptions does not provide an explain option, a ``cursor`` document must always be added to the ``aggregate`` command. Therefore ``useCursor`` is no longer needed. Removing ``useCursor`` is a backwards breaking change, so drivers should first deprecate this option in a minor release, and remove it in a major release.
```

Comment:
Some take-it-or-leave-it suggestions:

 * Replace "3.5.2" with "3.5.2+" or "3.5.2 and later".
 * Use "define" instead of "provide" in "does not provide an explain option"

Discussion in [DRIVERS-164](https://jira.mongodb.org/browse/DRIVERS-164) concluded that drivers don't need to add explain helpers (or support {{explain}} on aggregation), but it appears that some drivers do expose such an option per comments in [DRIVERS-375](https://jira.mongodb.org/browse/DRIVERS-375). In this case, we might want to add a sentence to address that possibility:

> If a driver does support an `explain` option, the `cursor` document should be omitted if `explain` is `true`.

Reply:
Those suggestions make sense. I reworded this a little to try to clarify that regardless of whether explain is provided, useCursor is no longer necessary as it will be known whether or not to add the cursor document.